static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js

```diff
@@ -33,32 +33,36 @@ define([
 
     return function () {
 
-        this.id = 'ContributionsEpicUsaCtaFakeNews';
+        this.id = 'ContributionsEpicUsaCta';
```

Comment:
We have already named a test with this name (https://github.com/guardian/frontend/pull/15031/), so we should give it a distinctive name so it's easy to analyse the results